<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1354</width>
    <height>832</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>10</pointsize>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="table_current">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>771</width>
      <height>271</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>10</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QTableWidget" name="table_condition">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>521</width>
      <height>211</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>4</number>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QTextEdit" name="txt_balance">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>130</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_code_del">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>190</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>선택종목 삭제</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>230</y>
      <width>461</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QComboBox" name="combo_buy"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>매도전략</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>선택종목 매수전략 clear</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="combo_sell"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>매수전략</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btn_sell_strategy_add">
       <property name="text">
        <string>선택종목에 매도전략 추가</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btn_buy_strategy_add">
       <property name="text">
        <string>선택종목에 매수전략 추가</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>선택종목 매도전략 clear</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>param: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_buy_param"/>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>param: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_sell_param"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>102</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>선택종목</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_condition_refresh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>test</string>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>조건식 refresh</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>계좌번호</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="combo_account"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>811</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="3">
      <widget class="QPushButton" name="btn_print_my_data">
       <property name="text">
        <string>data 확인</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_get_balance">
       <property name="text">
        <string>잔고조회 (보유종목)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="btn_test">
       <property name="text">
        <string>test</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_register_real_all">
       <property name="text">
        <string>전체 실시간 등록</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="toolTip">
        <string>현재가 갱신 되어 있어야 함</string>
       </property>
       <property name="text">
        <string>종목조회 (미보유종목)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QLineEdit" name="edit_code">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>종목코드</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_code">
       <property name="text">
        <string>종목추가</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_load_balance">
       <property name="text">
        <string>잔고파일 불러오기</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_save_balance">
       <property name="text">
        <string>잔고파일 저장</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1354</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPyStock">
    <property name="title">
     <string>PyStock</string>
    </property>
   </widget>
   <addaction name="menuPyStock"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_condition_refresh</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_condition_refresh_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>706</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_get_balance</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_balance_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>752</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_print_my_data</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_print_my_data_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>975</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>949</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_code</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_code_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_register_real_all</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_register_real_all_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combo_account</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_account_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>728</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_code_del</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_code_del_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>831</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>894</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_buy_strategy_add</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_buy_strategy_add_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>862</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>789</x>
     <y>471</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_sell_strategy_add</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_sell_strategy_add_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1249</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>1033</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_buy_strategy_clear_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>885</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>829</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_sell_strategy_clear_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1249</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>1019</x>
     <y>547</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_load_balance</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_load_balance_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_save_balance</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_save_balance_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>702</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>793</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_test</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_test_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1145</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>1249</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combo_buy</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_buy_strategy_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>969</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>1072</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combo_sell</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_sell_strategy_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1143</x>
     <y>308</y>
    </hint>
    <hint type="destinationlabel">
     <x>1284</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_interest_balance_btn_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_condition_refresh_btn_clicked()</slot>
  <slot>on_condition_result_btn_clicked()</slot>
  <slot>on_balance_btn_clicked()</slot>
  <slot>on_print_my_data_btn_clicked()</slot>
  <slot>on_code_btn_clicked()</slot>
  <slot>on_register_real_btn_clicked()</slot>
  <slot>on_register_real_all_btn_clicked()</slot>
  <slot>on_account_changed(QString)</slot>
  <slot>on_code_del_btn_clicked()</slot>
  <slot>on_buy_strategy_add_btn_clicked()</slot>
  <slot>on_sell_strategy_add_btn_clicked()</slot>
  <slot>on_buy_strategy_clear_btn_clicked()</slot>
  <slot>on_sell_strategy_clear_btn_clicked()</slot>
  <slot>on_load_balance_btn_clicked()</slot>
  <slot>on_save_balance_btn_clicked()</slot>
  <slot>on_test_btn_clicked()</slot>
  <slot>on_buy_strategy_changed(QString)</slot>
  <slot>on_sell_strategy_changed(QString)</slot>
  <slot>on_interest_balance_btn_clicked()</slot>
 </slots>
</ui>
